sunbeam_migrate/handlers/base.py

```diff
@@ -109,6 +109,22 @@ def get_supported_resource_filters(self) -> list[str]:
         """
         return []
 
+    def connect_member_resources_to_parent(
+        self,
+        parent_resource_id: str | None,
```

Comment:
I'd also pass the parent resource type. It simplifies lookups and furthermore, some resources may have more than one parent.